internal/api/auth/context.go

```diff
@@ -59,6 +59,10 @@ func VerifyTokenAndWriteCtxData(ctx context.Context, token, orgID string, t Toke
 	return context.WithValue(ctx, dataKey, CtxData{UserID: userID, OrgID: orgID, ProjectID: projectID, AgentID: agentID}), nil
 }
 
+func GetCtxDataKey() key {
```

Comment:
will be removed as soon as setup is mergef

Reply:
i think setup is merged

Reply:
yes i already did this